specs-go/config.go

```diff
@@ -22,11 +22,24 @@ type Spec struct {
 	Hooks Hooks `json:"hooks"`
 	// Annotations is an unstructured key value map that may be set by external tools to store and retrieve arbitrary metadata.
 	Annotations map[string]string `json:"annotations,omitempty"`
-
+	// KernelConfig specifies kernel-related configuration for virtual-machine-based runtimes.
+	KernelConfig KernelConfig `json:"kernelConfig"`
+	// ImagePath is the absolute path to the root filesystem image on the host which can be used by a virtual machine-based runtime.
+	ImagePath string `json:"imagePath"`
 	// Linux is platform specific configuration for Linux based containers.
 	Linux Linux `json:"linux" platform:"linux"`
```

Comment:
On Wed, May 18, 2016 at 04:32:53PM -0700, Tianon Gravi wrote:
> "LinuxContainer" would be redundant anyhow IMO given that this whole thing
> is "the container runtime spec" :smile:

Yeah, “linux-namespace-cgroup” or something would probably be better,
since those are the main components the current “linux” section is
based on.  I'm still not sure how much common ground there is to be
found with a big-tent approach that also covers VMs, although I guess
we'll see ;).